prdoc/pr_6169.prdoc

```diff
@@ -0,0 +1,19 @@
+title: Deprecate Gov_v1 treasury spend flow in `treasury` pallet
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Deprecates items from treasury pallet used in Gov_v1 flow.
+      Items deprecated: `spend_local`, `remove_approval`, `proposal_count`, `proposals`, `approvals`, `MaxApprovals`, `ProposalCount`, `Proposals`, `Approvals`.
```

Comment:
I do not think listing the names any helpful. Also do not mention Gov_v1, because `spend_local` could be used in a different ways, Gov_v1 is more Polkadot term. 

I would write that we deprecating the `spend_local` call and all associated items with it. Also I would add a migration instructions, under a title (### Migration ). There are at least two use cases to keep in mind. 1. users who used only `spend_local`. They need to setup `spend` to serve the same purpose. You can also change the configuration of treasury in `kitchensink-runtime` runtime and mention it additionally as an example. 2. Those who already use `spend` but for everything except `spend_local` , they need to configure `spend` also to serve the `spend_local use case`.

Reply:
kitchen sink runtime though configures the most general usage with multiple assets
```
	type AssetKind = u32;
	type Paymaster = PayAssetFromAccount<Assets, TreasuryAccount>;
```
is it a good idea to make usage of kitchensink runtime more narrow, because as I understand it's meant as an example of all the possibilities, and multiple asset configuration is the most flexible choice for treasury?

Reply:
Improved the prdoc